sdk/eventhubs/azure_messaging_eventhubs/CHANGELOG.md

```diff
@@ -4,8 +4,13 @@
 
 ### Features Added
 
+- Added EventHubs token refresh.
+- Added retries for failed EventHubs operations.
+
 ### Breaking Changes
 
+- The `fully_qualified_domain` parameter to the `open` method on the `ProducerClient` and `ConsumerClient` builder now takes an `&str` instead of a `String` to better follow the Rust API guidelines.
```

Comment:
Nit:
```suggestion
- The `fully_qualified_domain` parameter to the `open` method on the `ProducerClient` and `ConsumerClient` builder now takes an `&str` instead of a `String` to better follow the Azure SDK for Rust API guidelines.
```
Just to be clear who's guidelines we're talking about.